strkey/main_test.go

```diff
@@ -81,4 +89,16 @@ func TestIsValidEd25519SecretSeed(t *testing.T) {
 	invalidKey = "GDWZCOEQRODFCH6ISYQPWY67L3ULLWS5ISXYYL5GH43W7YFMTLB65PYM"
 	isValid = IsValidEd25519SecretSeed(invalidKey)
 	assert.Equal(t, false, isValid)
+
+	invalidKey = "SAV76USXIJOBMEQXPANUOQM6F5LIOTLPDIDVRJBFFE2MDJXG24TAPUU7="
+	isValid = IsValidEd25519PublicKey(invalidKey)
+	assert.Equal(t, false, isValid)
+
+	invalidKey = "SBCVMMCBEDB64TVJZFYJOJAERZC4YVVUOE6SYR2Y76CBTENGUSGWQABESI======"
+	isValid = IsValidEd25519PublicKey(invalidKey)
```

Comment:
This test needs to be testing the `IsValidEd25519SecretSeed` function.
```suggestion
	isValid = IsValidEd25519SecretSeed(invalidKey)
```